### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by MyRaceData**
edit (re-word) first paragraph to make the reason why a Java virtual machine is necessary and that installation of a Java virtual machine is a requirement of installing OpenHAB more clear

### CODE REVIEW

**Comment by MyRaceData** [comment]
I am sorry and am new at making changes to the documentation and also new to making pull request on github. Please clarify this sentence from the contributing to openHAB guidelines:
Commit messages must start with a capitalized and short summary (max. 50 chars) written in the imperative
What is the commit message and where / what is the imperative

**Comment by Confectrician** [comment]
>Commit messages must start with a capitalized and short summary

`Update index.md` is perfectly fine for now. 🙂 

Something like "re worded first paragraph" would be a bit better, since it describes the changes a bit.
But thats nothing to worry about.

Anyway you should read the `sign-off`part of the contribution guidelines.
Signing off is something that will be necessary for changes that are more than one line.

If you have additional questions, please ask.
I will do my best to answer them. 🙂 

BR
Jerome

File: installation/index.md [hunk 1]
```diff
@@ -7,9 +7,8 @@ title: Installation Overview
 
 # Installation Overview
 
-openHAB 2 is based on the Eclipse SmartHome framework and is fully written in Java.
-As such, it only depends on a Java Virtual Machine, which is available for many platforms.
-openHAB can be executed on different versions of **macOS** and **Windows** and many different variants of **Linux** (Ubuntu, Raspbian, ...).
+OpenHAB 2 is based on the Eclipse SmartHome framework which is fully written in Java. As such, OpenHAB requires a Java Virtual Machine to be installed on the system you intend to install openHAB on. A Java Vitual Machine is available for many platforms.
```
Comment (Confectrician) [inline installation/index.md]: Please change this part to one line per sentence.
The website build mechanism will take care of the design and we can track changes better, with only one sentence for each line.

**Comment by MyRaceData** [comment]
OK thanks for the tips. Others had explained the sign off process and imperative statements in another thread as well and I understand better and will try to use it in any further contributions. 
Also thanks for explanation of build mechanism.

**Comment by Confectrician** [comment]
Hey @MyRaceData,

You could have changed the sentence lines too, like you did it on the other pull request.

If you restore your patch-1 branch i can help you with that too, like we did in the other PR. 🙂 

image
You should have such a button above.

We can also talk about the sign-off part a bit more if you want some informatoin from me about it.

**Pull request closed without merging**